ui/src/timeMachine/components/view_options/DecimalPlaces.tsx

```diff
@@ -65,29 +70,23 @@ class DecimalPlacesOption extends PureComponent<Props, State> {
   }
 
   public handleSetValue = (e: ChangeEvent<HTMLInputElement>): void => {
-    const value = Number(e.target.value) || 0
+    const value = convertUserInputToNumOrNaN(e)
     const {digits, onDecimalPlacesChange} = this.props
 
-    if (value === null) {
-      onDecimalPlacesChange({digits, isEnforced: false})
+    if (value === value && value >= MIN_DECIMAL_PLACES) {
```

Comment:
do you think the logic here is worth testing?

Reply:
Definitely worth testing. I will cover this with e2e tests, which will be pushed once clockface releases 1.0.5

Reply:
So, it turns out that we can adjust the number of decimals, but we will not be able to see the an actual number with decimal places being updated to the correct decimal place in the e2e tests. But we do have tests to validate that the input field is in default or error status.